account_invoice_import_invoice2data/README.rst

```diff
@@ -92,6 +92,13 @@ Refer to the usage section of the module *account_invoice_import*.
    :alt: Try me on Runbot
    :target: https://runbot.odoo-community.org/runbot/226/8.0
 
+
+To use the invoice2data_templates addon, you need to:
```

Comment:
Similarly, don't mention `addon` but just say `To manage the templates in the database,`